persistence/src/main/java/edu/unc/lib/dl/persist/services/destroy/DestroyObjectsJob.java

```diff
@@ -125,8 +127,19 @@ public void run() {
                 }
 
                 if (!repoObj.getResource().hasProperty(RDF.type, Cdr.Tombstone)) {
+                    RepositoryObject parentObj = repoObj.getParent();
+
                     // purge tree with repoObj as root from repository
                     destroyTree(repoObj);
+
+                    // Add premis event to parent
+                    parentObj.getPremisLog().buildEvent(Premis.Deletion)
+                            .addAuthorizingAgent(agent.getUsername())
+                            .addOutcome(true)
+                            .addEventDetail("{0} object(s) were destroyed", deletedObjCount)
+                            .addEventDetail("Objects destroyed: {0}",
```

Comment:
there should probably be a newline after the colon so that all the ids have their own line